packages/apps/patterns/react-appkit/src/telemetry/base-properties.ts

```diff
@@ -3,13 +3,22 @@
 //
 
 import { Client } from '@dxos/client';
+import { log } from '@dxos/log';
 import { captureException } from '@dxos/sentry';
 import { humanize } from '@dxos/util';
 
 const IPDATA_API_KEY = process.env.IPDATA_API_KEY;
 
-export const DX_TELEMETRY = localStorage.getItem('halo-app:telemetry-disabled');
-export const DX_GROUP = localStorage.getItem('halo-app:telemetry-group');
+let DX_TELEMETRY: string | null = null;
+let DX_GROUP: string | null = null;
+try {
+  DX_TELEMETRY = localStorage.getItem('halo-app:telemetry-disabled');
```

Comment:
was this crashing? my impression is the localStorage.getItem returns null if the item isn't found? is this because localstorage isn't available in the extension?

Reply:
localStorage is throwing an error in sandboxed window in the extension